### ISSUE TITLE
app shits itself when going from vertical to horizontal monitor

### ISSUE THREAD

**Issue by Snapcicle** [ciderapp/Cider-2#961]
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ciderapp/Cider-2/issues) for a bug report that matches the one I want to file, without success.
- [x] I have confirmed that no [support FAQ](https://cider.sh/docs/client/faqs) resolves my issue.
- [x] I have checked that the correct behavior occurs on [Apple Music Web](https://music.apple.com) (if applicable).
- [x] I have verified that my issue is with a [supported version](https://github.com/ciderapp/Cider-2/blob/main/SECURITY.md) and that it is with the desktop app (Support for other areas can be found in our Discord / on the appropriate repo).

### How did you download Cider?

itch.io

### App Information

{"clientInfo":{"client":"dotnet","env":"production","arch":"AMD64","platform":"win32","platformFriendly":"Windows","build":"desktop","clientVersion":"1.0.0","capi":"874bdeb4-179a-4736-b04e-ea76fb9bfa0f"},"buildInfo":{"version":"2.6.1.4","commit":"d5cbb8ca86fca0bcb4fd064ab6456a80dc121a82","buildDate":"1739747806561","changelogUrl":"2.6.0-release"}}

### Operating System

Windows

### OS Version

windows 11

### How would you describe your issue?

Frontend

### Commit Hash

d5cbb8ca86fca0bcb4fd064ab6456a80dc121a82

### Describe the Bug

the app poops itseld when giong from vertical to horizontal when a song is playing and is in fullscreen

### Steps to Reproduce

vertical to horizontal when a song is playing and is in fullscreen

https://github.com/user-attachments/assets/7ee79ee7-de99-4fe9-8589-9cba746eaf90

**Comment by cryptofyre** [comment]
Cannot reproduce, likely fixed with upstream WV2 change internally as this is a V8 engine issue.
